lib/tasks/rubocop.rake

```diff
@@ -5,5 +5,5 @@ begin
     task.options = %w(--format simple)
   end
 rescue LoadError
-  puts 'Unable to load RuboCop.'
+  puts 'Unable to load RuboCop.' if Rails.env.development? || Rails.env.test?
```

Comment:
Classic @tingulfsen. :wink: 

This was deployed to -dev for a while.  Since rubocop is a development dependency, it is installed and loads fine.